cleanup/DeleteBasedonProperties/Readme.md

```diff
@@ -0,0 +1,23 @@
+Artifacts delete based on artifact properties User Plugin
+=====================================
+##Description:
+This plugin deletes builds based on properties defined on repositories . It can be run automatically as a scheduled job.
+
+##Features:
+This plugin Takes the Parameters from the Artifact/Repository
+
+It will scan all the repositories marked with property as `DataDeprecation'= 'true' `
+It will then look for property called `DataDeprecationMethod = 'version : x' ` 
```

Comment:
You probably don't need two properties. Just by checking property `DataDeprecationMethod` is enough to identify that that path must be cleaned. You should also change the property name to be in sync with the plugin and change the value to a single integer with the desired number of versions to keep. Suggestion: `cleanOldVersions.versionsToKeep = x`